backend/utils/const.js

```diff
@@ -90,6 +90,14 @@ const THEMES_CACHE = IS_TEST
   ? TEST_THEMES_CACHE
   : process.env.THEMES_CACHE || `${__dirname}/../themes`
 
+const DEFAULT_INFRA_RESOURCES = [
+  'gcp-files',
+  'gcp-dns',
+  'ipfs-pinata',
```

Comment:
super super nit: to follow the standard vendor-service used by the other entries, this should be named 'pinata-ipfs'

Reply:
This kind of bugs me too.  But if I change it then I'd have to do `ipfs-cluster-ipfs` which is ridiculous!  Or arguably it could be `ipfs-pinata-files` and `ipfs-cluster-files` but the deployment process between buckets and IPFS are separate enough that I don't want to do that either.  Maybe I should just give them all numbers and be done with it!

I could honestly hand-wring over this for days.

Reply:
haha. yeah, don't worry about. not worth overthinking this. lgtm as is.